tuf/client/client.go

```diff
@@ -88,7 +88,7 @@ func (c *Client) downloadRoot() error {
 		logrus.Debugf("Loading root with no expected checksum")
 
 		// get the cached root, if it exists, just for version checking
-		cachedRoot, _ := c.cache.GetMeta(role, -1)
+		cachedRoot, _ := c.cache.GetSized(role, -1)
```

Comment:
We're going to do a follow up to switch out anywhere that's using `GetSized` to just use `Get`, then the `NoSizeLimit` const becomes deprecated.